packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -1000,7 +998,6 @@ export default class GoogleCalendarService implements Calendar {
         googleChannelResourceId: googleChannelProps.resourceId,
         googleChannelResourceUri: googleChannelProps.resourceUri,
         googleChannelExpiration: googleChannelProps.expiration,
-        error,
```

Comment:
Instead of setting the error here, throw where the error comes and let the cron handle the error update at a single place.